src/App.js

```diff
@@ -1,11 +1,51 @@
-import React from 'react';
+import React, {useEffect, useState} from "react";
+import axios from "axios";
 import './App.css';
+import './components/PokemonCard';
+import PokemonCard from "./components/PokemonCard";
 
 function App() {
+    const [endpoint, setEndpoint] = useState(['https://pokeapi.co/api/v2/pokemon'])
+    const [pokemonData, setpokemonData] = useState([]);
```

Comment:
Zodra de setter aangeroepen wordt is `data` niet meer van het type array. Ik zou daarom de default waarde dan ook `undefined` maken
```suggestion
    const [pokemonData, setpokemonData] = useState();
```